### DISCUSSION TITLE
qn

### DISCUSSION THREAD

**Discussion by Kihuna544** [laravel/framework#52455] [Q&A] [answered]
hellow guys,hope we are all using latest version of laravel..i am tryng to add some code in the kernel.php and dont see it in my project foflders,can somebody help me with this please

**Comment by noefleury** [comment, marked as answer]
Hello,

From Laravel 11, Kernel files disappeared and the logic moved to the `bootstrap/app.php` file.

From the release note :

> The bootstrap/app.php file has been revitalized as a code-first application configuration file. From this file, you may now customize your application's routing, middleware, service providers, exception handling, and more.

Details [following this link](https://laravel.com/docs/11.x/releases).
